Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -14,8 +14,8 @@ private let DefaultSuggestedSitesKey = "topSites.deletedSuggestedSites"
 
 // MARK: -  Lifecycle
 struct FirefoxHomeUX {
-    static let rowSpacing: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 30 : 20
-    static let highlightCellHeight: CGFloat = UIDevice.current.userInterfaceIdiom == .pad ? 250 : 200
+    static let rowSpacing: CGFloat = UIDevice.current.isPad ? 30 : 20
+    static let highlightCellHeight: CGFloat = UIDevice.current.isPad ? 250 : 200
```

Comment:
I suggest replacing this with a UITraitCollection based approach. You could be on an iPad, but in split screen mode.